browser/ui/speedreader/speedreader_tab_helper.h

```diff
@@ -187,8 +186,11 @@ class SpeedreaderTabHelper
 
   DistillState distill_state_{DistillStates::ViewOriginal()};
 
+  raw_ref<SpeedreaderService> speedreader_service_;
+
   const raw_ptr<SpeedreaderRewriterService> rewriter_service_ =
       nullptr;  // NOT OWNED
```

Comment:
unrelated, but why do we have a `NOT OWNED` commend on a `raw_ptr`?

Reply:
probably it was converted from a regular pointer and not removed?